plugin/signature_help.py

```diff
@@ -167,3 +188,59 @@ def _build_popup_content(self) -> str:
         if sigDocs:
             formatted.append(sigDocs)
         return "\n".join(formatted)
+
+    def _build_popup_content_style_vscode(self) -> str:
+        # Fetch all the relevant data.
+        signature = {}  # type: Dict[str, Any]
+        signature_label = ""
+        signature_documentation = ""
+        parameter = {}  # type: Dict[str, Any]
```

Comment:
`signature` and `parameter` are not used outside their respective if blocks below, no need to declare them here?